src/Loop.h

```diff
@@ -79,6 +80,8 @@ struct Loop {
         /* We also need some timers (should live off the one 4 second timer rather) */
         LoopData *loopData = (LoopData *) us_loop_ext((struct us_loop_t *) loop);
         loopData->dateTimer = us_create_timer((struct us_loop_t *) loop, 1, sizeof(LoopData *));
+        loopData->updateDate();
```

Comment:
This was to ensure we always have a date set

Reply:
It's set in the constructor